cpp/benchmarks/io/orc/orc_writer_benchmark.cpp

```diff
@@ -113,6 +119,6 @@ BENCHMARK_DEFINE_F(OrcWrite, writer_options)
 BENCHMARK_REGISTER_F(OrcWrite, writer_options)
   ->ArgsProduct({{int32_t(cudf::io::compression_type::NONE),
                   int32_t(cudf::io::compression_type::SNAPPY)},
-                 {0, 1}})
+                 {0, 1, 2}})
```

Comment:
Why not `{int32_t(cudf::io::STATISTICS_NONE), ...}`

Reply:
Fixed. This reads much better now.